### PR TITLE
Changed color of input to #ffaf4b on focus

### PR DESCRIPTION
**Pull request by jayaike**
#### Changes proposed in this pull request:

- Changed color of input to #ffaf4b on focus as requested

#### Screenshots below

- Screenshot (119)
- Screenshot (118)

### CODE REVIEW

**Comment by lunayach** [comment]
Hi @nsjcorps, Thanks a lot for the PR. Can you edit the PR description with the screenshot of the UI changes?

File: src/app/components/utility/input/input.component.scss [hunk 1]
```diff
@@ -130,6 +130,7 @@
     &:focus {
       outline: none;
       border-bottom: 1px solid $yellow;
+      box-shadow:none;
     }
```
Comment (lunayach) [inline src/app/components/utility/input/input.component.scss:134]: @nsjcorps Where is the colour changed?

Comment (jayaike) [reply src/app/components/utility/input/input.component.scss:134]: The color is defined in the $yellow variable as #ffaf4b.

Comment (jayaike) [reply src/app/components/utility/input/input.component.scss:134]: It is defined in the _variables.scss file as #ffaf4b.
Screenshot (120)

File: src/app/components/utility/input/input.component.scss [hunk 2]
```diff
@@ -130,6 +130,7 @@
     &:focus {
       outline: none;
       border-bottom: 1px solid $yellow;
+      box-shadow:none;
```
Comment (Ram81) [inline src/app/components/utility/input/input.component.scss]: @nsjcorps please add a space after `:`

**Review by Ram81 — CHANGES_REQUESTED** [review]

File: src/app/components/utility/input/input.component.scss (hunk 2 above)
Comment (jayaike) [reply src/app/components/utility/input/input.component.scss]: Alright. Will do that now.

File: .gitignore [hunk 3]
```diff
@@ -37,6 +37,8 @@ npm-debug.log
 testem.log
 /typings
 package-lock.json
+yarn.lock
+yarn-error.log
```
Comment (Sanji515) [inline .gitignore]: These changes are not related to this PR, I think we should revert these changes.

**Review by Sanji515 — COMMENTED** [review]
Rest LGTM :+1:

**Review by Sanji515 — APPROVED** [review]
LGTM! It can be merged :tada:

**Review by Ram81 — APPROVED** [review]

**Review by RishabhJain2018 — APPROVED** [review]
